src/remote.ts

```diff
@@ -126,6 +127,9 @@ export class Remote {
       this.registerLabelFormatter(remoteAuthority, this.storage.workspace.owner_name, this.storage.workspace.name),
     )
 
+    // Initialize any WorkspaceAction notifications (auto-off, upcoming deletion)
+    const action = await WorkspaceAction.init(this.vscodeProposed, this.storage)
```

Comment:
I think that makes a lot of sense with, say, an auto-stop notification and we might want to tweak it in the future, especially as I don't want to bombard users with messaging.
The reason we didn't scope notifications more narrowly for this first pass is because we want workspace owners to be informed about inactive workspaces that are about to be locked or deleted (locked status coming in follow-up PR). Naturally, they won't be using these workspaces; they will be inactive.
Down the road, it would be nice to have users be able to toggle notification settings for specific workspaces or turn them off entirely. Not sure how accommodating the VS Code API will be here.

Reply:
Makes sense!